akka-stream/src/main/scala/akka/stream/scaladsl/FileIO.scala

```diff
@@ -53,9 +53,10 @@ object FileIO {
    *
    * @param f         the file path to read from
    * @param chunkSize the size of each read operation, defaults to 8192
+   * @param startPosition the start position to read from, defaults to 0
    */
-  def fromPath(f: Path, chunkSize: Int = 8192): Source[ByteString, Future[IOResult]] =
-    new Source(new FileSource(f, chunkSize, DefaultAttributes.fileSource, sourceShape("FileSource")))
+  def fromPath(f: Path, chunkSize: Int = 8192, startPosition: Long = 0): Source[ByteString, Future[IOResult]] =
```

Comment:
This won't be binary compatible, so we need to do it the same way as with the Java API